lib/index.js

```diff
@@ -12,11 +12,11 @@ const contextModules = [
 
 for(const module of contextModules) {
   const {contexts, constants: contextConstants} = require(module);
-  for(const c in contextConstants) {
-    if(c.includes('URL')) {
-      const contextUrl = contextConstants[c];
-      bedrockConstants[c] = contextUrl;
-      jsonLdDocumentLoader.addStatic(contextUrl, contexts.get(contextUrl));
-    }
+  for(const [contextUrl, context] of contexts) {
+    jsonLdDocumentLoader.addStatic(contextUrl, context);
+  }
+  for(const contextName in contextConstants) {
+    const contextValue = contextConstants[contextName];
+    bedrockConstants[contextName] = contextValue;
```

Comment:
```suggestion
    bedrockConstants[contextName] = contextConstants[contextName];
```

Reply:
This is shorter -- and more accurate given that the values aren't always "context values".